PageObjectPattern/src/test/java/shopping/ResetPasswordTest.java

```diff
@@ -0,0 +1,21 @@
+package shopping;
+
+import static org.junit.Assert.assertTrue;
+import org.junit.Test;
+
+public class ResetPasswordTest extends MainTestBase {
+
+	@Test
+	public void checkPasswordReset() {
+		driver.get(baseUrl + "/index.php?controller=authentication&back=my-account");
```

Comment:
Better to explicitly click on the link in this case rather than use the direct url.

Reply:
done